plugins/libplugin-pay.c

```diff
@@ -1254,12 +1278,21 @@ static void payment_compute_onion_payloads(struct payment *p)
 		 * i+1 */
 		payment_add_hop_onion_payload(p, &cr->hops[i], &p->route[i],
 					      &p->route[i + 1], false, NULL);
+		tal_append_fmt(&routetxt, "%s -> ",
+			       type_to_string(tmpctx, struct short_channel_id,
+					      &p->route[i].channel_id));
 	}
 
 	/* Final hop */
 	payment_add_hop_onion_payload(
 	    p, &cr->hops[hopcount - 1], &p->route[hopcount - 1],
 	    &p->route[hopcount - 1], true, root->payment_secret);
+	tal_append_fmt(&routetxt, "%s",
+		       type_to_string(tmpctx, struct short_channel_id,
+				      &p->route[hopcount - 1].channel_id));
+
+	paymod_log(p, LOG_DBG,
+		   "Created outgoing onion for route: %s", routetxt);
```

Comment:
Nice change, this should help alot with debugging (and is way easier to grok than the `getroute` output). We might even want to keep this short format around attached to the `struct payment` to add context to later logs.